config.toml

```
@@ -99,14 +99,14 @@ relativeURLs = false
     weight = 2
 [[menu.main]]
     parent = "epidemic-signals"
-    name = "Signal Discovery & Selection"
+    name = "Indicators Discovery & Selection"
     url = "/signals/"
     weight = 3
     [menu.main.params]
         external = true
 [[menu.main]]
     parent = "epidemic-signals"
-    name = "Signal Visualizaion"
+    name = "Indicators Visualizaion"
```

preserve plurality & fix old typo
```suggestion
    name = "Indicator Visualization"
```